sdk/python/feast/feature_store.py

```diff
@@ -254,6 +255,23 @@ def apply(
             partial=True,
         )
 
+    @log_exceptions_and_usage
+    def teardown(self):
+        feature_defs: List[Union[FeatureView, FeatureTable]] = []
```

Comment:
Should we call this `tables`? That is what we're currently calling it elsewhere.